src/backend/direct.rs

```diff
@@ -867,7 +868,23 @@ impl crate::Context for Context {
         wgc::span!(_guard, TRACE, "Device::create_bind_group wrapper");
         use wgc::binding_model as bm;
 
-        let texture_view_arena: Arena<wgc::id::TextureViewId> = Arena::new();
+        let mut arrayed_texture_views = Vec::new();
```

Comment:
It kinda pains me to allocate this without a `with_capacity` on the known max size, but it's good enough for now.

Reply:
Agreed! There are things to try, but they involve `unsafe` one way or the other.... Let's leave for a follow-up